### PR TITLE
Bump version to 1.2.0-pre

### PR DESCRIPTION
**Pull request by nkuba**
We bump version to 1.2.0-pre.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@keep-network/sortition-pools",
-  "version": "1.1.2",
+  "version": "1.2.0-pre.0",
```
Comment (pdyraga) [inline package-lock.json:3]: The most recent version published to NPM is `1.2.0-pre.4`. Should it be reflected here?

Comment (nkuba) [reply package-lock.json:3]: `pre` versions for publication get auto-bumped by the publication script. As we're not updating the version in `package.json` in the repo I'm not sure if we want to update it here.

**Comment by lukasz-zimnoch** [comment]
@nkuba can you rebase on master and solve the conflicts?

**Comment by nkuba** [comment]
@lukasz-zimnoch it's ready.

**Review by lukasz-zimnoch — APPROVED** [review]

**Pull request merged**